test/functional/cmdLineTests/criu/criu_nonPortable.xml

```diff
@@ -1045,4 +1046,20 @@
     <output type="failure" caseSensitive="yes" regex="no">Could not dump the JVM processes, err=-70</output>
     <output type="failure" caseSensitive="yes" regex="no">User requested Java dump using</output>
   </test>
+
```

Comment:
we should have a separate file for crac tests, as crac will not be enable by default initially

Reply:
Added `criu_nonPortable_jdk_crac.xml`.

Currently `CRAC_SUPPORT` is enabled by default at `xa64` platform - https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/91440dd91fc4020ae57da00c9bdeea3a4f730ea6/closed/autoconf/custom-hook.m4#L354-L357